### PR TITLE
78 starts-with

### PR DESCRIPTION
**Pull request by levBagryansky**
Implemented `text.starts-with` via `index-of.check-if-starts-from-index`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@EugeneDar please review

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -60,11 +60,11 @@
   # Checks that string ends with substr
   [substr] > ends-with /bool
 
-  # Checks that string starts with substr
```
Comment (EugeneDar) [inline src/main/eo/org/eolang/txt/text.eo:63]: @levBagryansky why you removed description?

**Review by EugeneDar — CHANGES_REQUESTED** [review]
@levBagryansky looks good for me, just one comment above.

File: src/main/eo/org/eolang/txt/text.eo (hunk 1 above)
Comment (levBagryansky) [reply src/main/eo/org/eolang/txt/text.eo:63]: @EugeneDar my bad

**Review by EugeneDar — APPROVED** [review]
@levBagryansky looks good now.
@yegor256 please check.

**Comment by EugeneDar** [comment]
@levBagryansky I've already approved it.

**Review by EugeneDar — APPROVED** [review]
@levBagryansky I've already approved it.

**Comment by levBagryansky** [comment]
@yegor256 please look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**